src/peripherals/peripheral.ts

```diff
@@ -31,15 +31,15 @@ export class BasePeripheral implements Peripheral {
   constructor(protected rp2040: RP2040, readonly name: string) {}
 
   readUint32(offset: number) {
-    console.warn(`Unimplemented peripheral ${this.name} read from ${offset.toString(16)}`);
+    this.warn(`Unimplemented peripheral ${this.name} read from ${offset.toString(16)}`);
```

Comment:
optional, but now we can remove the peripheral name from the warning message (the logger will display it anyway)

same for the warning in writeUint32()

Reply:
Done